src/IssueViz.Security/HotspotsControl/HotspotsControl.xaml.cs

```diff
@@ -18,19 +18,45 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System;
 using System.Windows.Controls;
+using Microsoft.Internal.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Shell.TableManager;
+using SonarLint.VisualStudio.IssueVisualization.Security.HotspotsControl.TableDataSource;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsControl
 {
-    internal sealed partial class HotspotsControl : UserControl
+    internal sealed partial class HotspotsControl : UserControl, IDisposable
```

Comment:
Using `HotspotsControl` in the namespace and the type is a bit awkward. Could one of them be changed? e.g. HotspotsTable, HotspotsList, TableControl,...